lizard_connector/connector.py

```diff
@@ -192,8 +199,8 @@ def download(self, *querydicts, **queries):
         q = lizard_connector.queries.QueryDictionary(page_size=self.max_results
                                                      )
         q.update(*querydicts, **queries)
-        query = "?" + urllib.parse.urlencode(q)
-        url = urllib.parse.urljoin(self.base_url, query)
+        query = "?" + urllib_parse.urlencode(q)
```

Comment:
urlencode zit niet in python 2.7 urlparse maar in urllib. Je zal urlencode dus los moeten importeren voor beide python versies.